components/Panel/StakeUnstakePanel/CooldownTimer.tsx

```diff
@@ -8,20 +8,22 @@ import styled from "styled-components";
 interface Props {
   cooldownEnds: Date | null | undefined;
   pendingUnstake: BigNumber | undefined;
-  canClaim: boolean;
-  onClaim: () => void;
+  isReadyToUnstake: boolean;
```

Comment:
I've removed "claim" wording from unstake components to prevent confusion with claiming rewards